Tiltfile

```diff
@@ -0,0 +1,175 @@
+# -*- mode: Python -*-
+
+# set defaults
+
+settings = {
+    "enable_providers": ["docker"],
+}
+
+# global settings
+settings.update(read_json(
+    "tilt-settings.json",
+    default = {},
+))
+
+allow_k8s_contexts(settings.get("allowed_contexts"))
+
+default_registry(settings.get("default_registry"))
+
+providers = {
+    "core": {
+        "context": ".",
+        "image": "gcr.io/k8s-staging-cluster-api/cluster-api-controller",
+        "live_reload_deps": [
+            "main.go",
+            "go.mod",
+            "go.sum",
+            "api",
+            "bootstrap",
+            "cmd",
+            "controllers",
+            "controlplane",
+            "errors",
+            "third_party",
+            "util",
+        ],
+    },
+    "docker": {
```

Comment:
Shouldn’t this be in the other repo for v1a2?

Reply:
Yes, it's close to EOD and I was doing this quickly... will open a separate PR